source/CustomNode.h

```diff
@@ -75,6 +75,7 @@ class CustomNode : public cugl::Node {
 						   onScreenAngle <= -globals::SEG_CUTOFF_ANGLE);
 	}
 
+   protected:
```

Comment:
Why are you making these private methods protected? The child classes really aren't supposed to have to handle any of this spawn positioning code at all.

Reply:
I needed to expose this to be able to let `breachNode` move its `sparkleNode` around. I found a workaround tho so I guess I can undo this.